src/components/portfolios/PortfolioGrid.jsx

```diff
@@ -0,0 +1,19 @@
+import PortfolioCard from "./PortfolioCard";
+import SkeletonCard from "../common/atoms/SkeletonCard";
+
+const PortfolioGrid = ({ portfolios, isFetching }) => {
+  return (
+    <div className="portfolio-grid grid grid-cols-2 gap-[11px] py-[11px] px-[11px] w-full">
+      {portfolios?.map((portfolio) => (
+        <PortfolioCard key={portfolio.id} portfolio={portfolio} />
+      ))}
+      {isFetching && (
+        <>
+          <SkeletonCard /> <SkeletonCard />
+        </>
+      )}
```

Comment:
의미적으로 isFetching이면 스켈레톤, 아니면 카드를 렌더링하는 것 같아요
이 2개 경우가 동시에 일어날 수는 없을 것 같아서 여기는 삼항 연산자가 더 어울릴 것 같습니다!

```suggestion

      {isFetching ? (
        <>
          <SkeletonCard /> <SkeletonCard />
        </> :
        portfolios?.map((portfolio) => (
          <PortfolioCard key={portfolio.id} portfolio={portfolio} />
        ))
      )}
```

Reply:
수정했습니다!!

Reply:
이 부분은 삼항 연산자로 수정하게 된다면 의도하던대로 스켈레톤이 나오지 않을 것 같습니다! isFetching 일 때 렌더링된 카드 아래에 스켈레톤이 나오는 게 맞습니다:)

Reply:
맞네요.! isFetching은 캐시된 영역까지 스켈레톤으로 렌더링 되는 문제가 생기군요.